.github/workflows/tests.yml

```diff
@@ -32,13 +32,15 @@ jobs:
           - {impl: python, python: "3.10", postgres: "postgres:13"}
           - {impl: python, python: "3.11", postgres: "postgres:14"}
           - {impl: python, python: "3.12.0-beta.4 - 3.12", postgres: "postgres:15", libpq: oldest}
+          - {impl: python, python: "3.12.0-beta.4 - 3.12", postgres: "postgres:15", libpq: master}
 
           - {impl: c, python: "3.7", postgres: "postgres:15", libpq: newest}
           - {impl: c, python: "3.8", postgres: "postgres:13"}
           - {impl: c, python: "3.9", postgres: "postgres:14"}
           - {impl: c, python: "3.10", postgres: "postgres:12"}
           - {impl: c, python: "3.11", postgres: "postgres:11", libpq: oldest}
           - {impl: c, python: "3.12.0-beta.4 - 3.12", postgres: "postgres:10", libpq: newest}
+          - {impl: c, python: "3.12.0-beta.4 - 3.12", postgres: "postgres:10", libpq: master}
 
```

Comment:
Instead of adding another axis, maybe update an existing one so same some CI runtime?

Reply:
done

Reply:
@JelteF Amazing, really appreciated!

I've cleaned up your history and a bit and rewritten commit messages to avoid inflicting too much pedantry on you. Will merge this to master soon.